### PR TITLE
Tweak behaviour analysis

### PR DESCRIPTION
**Pull request by dneto0**
- statement sequence doesn't always have a semicolon between statements
  e.g. this lets s1 be switch, if, loop, for statements.

Editorial:
- Fix phrasing to 'f has behaviour B'.
- Add "algorithm" class info for each behaviour analysis row.
  This lets the reader click on placeholders to colour-code them in the
  other places in a row.

### CODE REVIEW

File: wgsl/index.bs
Comment (RobinMorisset) [inline wgsl/index.bs:5779]: Nit: I used the "behavior" spelling throughout instead of "behaviour", it should probably be kept consistent.

File: wgsl/index.bs
Comment (RobinMorisset) [inline wgsl/index.bs:5782]: I just noticed that this is `B` instead of `|B|`. I believe the latter would be better, and would allow removing B from the ignored vars list.

File: wgsl/index.bs
Comment (RobinMorisset) [inline wgsl/index.bs:15]: I don't see any row where s1/s2/sn are defined and not used. Why do they have to be marked Ignored?

**Review by RobinMorisset — APPROVED** [review]
Thanks for these fixes!
LGTM with a couple of nitpicks.

File: wgsl/index.bs
Comment (dneto0) [reply wgsl/index.bs:5779]: Before your PR, there were 13 "behaviour" instances and 1 "behavior" instance. :-)

File: wgsl/index.bs
Comment (dneto0) [reply wgsl/index.bs:15]: I added cases for the ones that errored out.  I'll check again.

Comment (dneto0) [reply wgsl/index.bs]: It occurs because of the rowspans.  The second row of the rowspans for `loop` and `switch` only have one instance of |s1 and |s2| (for loop), and |s1| and |sn| (for switch)

```
bikeshed --die-on=everything spec index.bs
WARNING: The following <var>s were only used once in the document:
  's1', in algorithm 'loop with continuing with break behavior'
  's2', in algorithm 'loop with continuing with break behavior'
  's1', in algorithm 'switch with break behavior'
  'sn', in algorithm 'switch with break behavior'
If these are not typos, please add an ignore='' attribute to the <var>.
 ✘  Did not generate, due to fatal errors
```

File: wgsl/index.bs
Comment (dneto0) [reply wgsl/index.bs]: Taking your suggestion (for these new instances)

File: wgsl/index.bs
Comment (RobinMorisset) [reply wgsl/index.bs]: got it, thanks

**Comment by RobinMorisset** [comment]
Feel free to land this, unless you are waiting for another review.

**Comment by dneto0** [comment]
> Feel free to land this, unless you are waiting for another review.

I was waiting for the bots.  :-)

**Pull request merged**
